fpga_interchange/fasm_generators/xc7.py

```diff
@@ -377,6 +539,12 @@ def handle_luts(self):
 
             self.luts[key][LutsEnum.from_str(lut_type)] = phys_lut_init
 
+    def handle_lutram(slef):
```

Comment:
This is not related to RAMB36 so please remove it

Reply:
Removed it form PR